lib/crypto/eosdart/src/client.dart

```diff
@@ -222,10 +222,21 @@ class EOSClient {
   }
 
   /// Get Key Accounts
```

Comment:
This change cannot go in, because getAccountsByKey isn't working @chuck-h

Reply:
I reversed this back in the light wallet code at some point the get_key_accounts is working, but get_accounts_by_authorizers is not. 

I am guessing has to do with Antelope changes.

Reply:
It has been working for me since late April when I pulled it into my dev tree. What failures were you seeing?
![image](https://github.com/JoinSEEDS/seeds_light_wallet/assets/2141014/f7148d08-5aad-47fd-ac6f-d343fb5acdc6)

Reply:
OK, reversed this at b0c0df6a6f65 . But I don't understand why. @n13

Reply:
Thanks. I don't know but it totally failed for me for some time.... IDK why.